src/App.jsx

```diff
@@ -3,18 +3,28 @@ import React, {useState, useEffect} from "react";
 export const App = () => {
 
     const [value, setValue] = useState('')
-    const [responseData, setResponseData] = useState('')
+    const [responseData, setResponseData] = useState([])
 
-    useEffect(async () => {
-        const response = await fetch('http://localhost:8008/albums')
-        const data = await response.json()
-        setResponseData(data)
-    })
+    useEffect( () => {
+        const getData = async () => {
+            const response = await fetch('http://localhost:8008/albums')
+            const data = await response.json()
+            const datafilter = data.filter((x) => {
```

Comment:
Filtering of the records should be on the BE side. FE should just send search value, and BE should respond with filtered records.